pkgs/by-name/wh/wheelwizard/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  buildDotnetModule,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  makeWrapper,
+  makeDesktopItem,
+  desktop-file-utils,
```

Comment:
Another minor nit, can you order these imports in a different way?

It might make it a bit easier to follow to add new deps to if we need to.